trimChamber.py

```diff
@@ -4,54 +4,64 @@
 By: Christine McLean (ch.mclean@cern.ch), Cameron Bravo (c.bravo@cern.ch), Elizabeth Starling (elizabeth.starling@cern.ch)
 """
 
-import sys, os, random, time
+import sys
 from array import array
-from GEMDAQTestSuite import *
+from macros.fitScanData import fitScanData
 from gempython.tools.vfat_user_functions_uhal import *
-from fitScanData import fitScanData
 
 from qcoptions import parser
 
 parser.add_option("--path", type="string", dest="path", default="data",
                   help="Specify Output File Path", metavar="path")
+parser.add_option("-p", "--ptrim", type="float", dest="ptrim", default=4.0,
+                  help="Specify GEB (long/short)", metavar="GEBtype")
 
```

Comment:
The help does not reflect this option parameter.
Also, suggest removing the single letter ```-p``` flag for future considerations